<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="StationText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>741</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="SystemText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>741</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>731</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="SysNamAutoClipboard">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>System Name Auto Clipboard</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="AlwaysOnTop">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Always On Top</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DoneStations">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>120</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Done!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClipboardSystemName">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>100</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clipboard System Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClipboardStationName">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>140</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clipboard Station Name</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>10</y>
      <width>401</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="Previous">
       <property name="text">
        <string>Previous</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Next">
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RedItOut">
       <property name="text">
        <string>Red It Out</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="AddStation">
       <property name="text">
        <string>Add Station</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Ignore">
       <property name="text">
        <string>Ignore</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>111</y>
      <width>82</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="LandPadSize">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Landing Pad Size</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="PadUnknown">
       <property name="text">
        <string>Unknown</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="PadMedium">
       <property name="text">
        <string>Medium</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="PadLarge">
       <property name="text">
        <string>Large</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>104</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="DistanceFromStar">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Distance from Star Ls</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="DistanceToStarLs">
       <property name="maximum">
        <number>9999999</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>81</width>
      <height>87</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="HasMarket">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Market</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="MarketUnknown">
       <property name="text">
        <string>Unknown</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="MarketYes">
       <property name="text">
        <string>Yes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="MarketNo">
       <property name="text">
        <string>No</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_CSV"/>
    <addaction name="actionSave_CSV"/>
    <addaction name="actionLoad_Red_List"/>
    <addaction name="actionSave_Red_List"/>
    <addaction name="actionLoad_Ignore_List"/>
    <addaction name="actionSave_Ignore_List"/>
    <addaction name="actionSave_Done_Stations"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad_CSV">
   <property name="text">
    <string>Load CSV</string>
   </property>
  </action>
  <action name="actionSave_CSV">
   <property name="text">
    <string>Save CSV</string>
   </property>
  </action>
  <action name="actionLoad_Red_List">
   <property name="text">
    <string>Load Red List</string>
   </property>
  </action>
  <action name="actionSave_Ignore_List">
   <property name="text">
    <string>Save Ignore List</string>
   </property>
  </action>
  <action name="actionSave_Red_List">
   <property name="text">
    <string>Save Red List</string>
   </property>
  </action>
  <action name="actionLoad_Ignore_List">
   <property name="text">
    <string>Load Ignore List</string>
   </property>
  </action>
  <action name="actionSave_Done_Stations">
   <property name="text">
    <string>Save Done Stations</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
